### PR TITLE
Update Go version to 1.20 

### PR DESCRIPTION
**Pull request by tarakby**
Update Go version to 1.20

### CODE REVIEW

File: utils/unittest/fixtures.go [hunk 1]
```diff
@@ -417,7 +417,7 @@ func BlockHeaderFixture(opts ...func(header *flow.Header)) *flow.Header {
 
 func CidFixture() cid.Cid {
 	data := make([]byte, 1024)
-	_, _ = rand.Read(data)
+	_, _ = crand.Read(data)
```
Comment (peterargue) [inline utils/unittest/fixtures.go:420]: it'd be best not to swallow this error. Maybe update the method either to accept a testing object and require no error, or just return the error. It doesn't look like this fixture is even used.

**Review by peterargue — APPROVED** [review]
Added one comment, but otherwise looks good

File: utils/unittest/fixtures.go (hunk 1 above)
Comment (tarakby) [reply utils/unittest/fixtures.go:420]: I would rather delete non-used functions. Thanks for the flag!

Comment (tarakby) [reply utils/unittest/fixtures.go:420]: At the same time, for fixtures I think failing randomness would just lead to the byte array remaining all zeros, which is still a valid "fixture".

**Comment by tarakby** [comment]
bors merge

**Pull request merged**